src/openvic-simulation/utility/StringUtils.hpp

```diff
@@ -160,6 +160,22 @@ namespace OpenVic::StringUtils {
 		return std::equal(lhs.begin(), lhs.end(), rhs.begin(), rhs.end(), ichar_equals);
 	}
 
+	inline constexpr std::string string_tolower(std::string_view str) {
```

Comment:
Wouldn't to_lower suffice?

Reply:
Do you mean `godot::String`'s `to_lower` function? That's not available in the SIM submodule, all we have access to here is `std::tolower` which only converts a single character